### PR TITLE
added homework1

### PR DESCRIPTION
**Pull request by VitaliySokoliuk**

### CODE REVIEW

File: homework1/test/java/org/example/CalculationTest.java [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+        calculation = null;
+        System.out.println("After test");
+    }
+
+    @Test
+    public void testBonus() {
+        int actual = calculation.bonus();
+        int expected = 100;
+        Assert.assertEquals(actual, expected);
+        System.out.println("Test 'testBonus'");
```
Comment (VolodymyrTymtsiv) [inline homework1/test/java/org/example/CalculationTest.java]: It doesn't provide any useful info as the name of the test anyway will be displayed in results/console.
So no need to ass this `System.out`.

Comment (VitaliySokoliuk) [reply homework1/test/java/org/example/CalculationTest.java]: All this `System.out.println() ` each for better understanding of the order of calling annotations and methods.

Comment (VitaliySokoliuk) [reply homework1/test/java/org/example/CalculationTest.java]: I can merge and pull request?

Comment (VolodymyrTymtsiv) [reply homework1/test/java/org/example/CalculationTest.java]: Yes, when you see the following sign you can merge
image

Comment (VitaliySokoliuk) [reply homework1/test/java/org/example/CalculationTest.java]: okay

**Pull request merged**